pyvista/utilities/features.py

```diff
@@ -172,6 +172,42 @@ def transform_vectors_sph_to_cart(theta, phi, r, u, v, w):
     return u_t, v_t, w_t
 
 
+def cart_to_sphe(x, y, z):
```

Comment:
I don't love this name... I'd prefer a more verbose `cartesian_to_spherical`

Reply:
Changed to `cartesian_to_spherical` 9811bbf.